content/api/commands/writefile.md

```diff
@@ -206,10 +207,10 @@ cy.readFile(filename, null).then((obj) => {
 
 ### Requirements [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Chains-of-Commands)
 
-- `cy.writeFile()` requires being chained off of `cy`.
-
-- `cy.writeFile()` requires the file be successfully written to disk. Anything
-  preventing this such as OS permission issues will cause it to fail.
+<List><li>`cy.writeFile()` requires being chained off of
+`cy`.</li><li>`cy.writeFile()` requires the file be successfully written to
+disk. Anything preventing this such as OS permission issues will cause it to
+fail.</li></List>
```

Comment:
what's this `<List>` stuff all about?

Reply:
Good question. Some files use markdown lists, others use this <List> component. Some use both! I was shooting for consistency, but I could be consistent in the other direction. Both seem to produce the same markup.

Reply:
There's a [transform](https://github.com/cypress-io/cypress-documentation/blob/master/scripts/transform.js#L654) script that seems to have injected these Lists during the migration to Nuxt. It might be okay to forgo those at this point and use markdown syntax directly? It's definitely more readable that way.

Reply:
Yeah, I prefer the markdown syntax. Maybe <List> was easier for the purposes of auto-transformation, but we no longer need to use it?

Reply:
Yah, I have no idea why some have the List elements. Maybe they don't render correctly in some instances of the regular markdown list.